app/controllers/speech_controller.rb

```diff
@@ -0,0 +1,5 @@
+class SpeechController < ContentItemsController
+    def show
+        @presenter =  SpeechPresenter.new(@content_item)
```

Comment:
The presenter and view hasn't been included yet, so if you were to load a speech page whilst on this commit it would error.
You could update this commit so that the `show` method is empty, and also include an empty view (app/views/speech/show.html.erb).